app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,70 @@
+<header>
+  <%= link_to groups_path, data: { turbo: false }  do %>
+    <%= inline_svg_tag "chevron_left.svg", class: "chevron" %>
+  <% end %>
+  <h1>transactions</h1>
+  <div id="whatever">
+    <%= inline_svg_tag "menu.svg", class: "burger-menu" %>
+  </div>
+</header>
+
+<main class="with_actions">
+  <% flash.each do |name, msg| -%>
+    <%= content_tag :div, msg, class: name %>
+  <% end -%>
+
+  <div class='top_category'>
+    <%= link_to edit_group_url(@group), data: { turbo: false }, class: 'item' do %>
+      <%= image_tag(check_url(@group.icon), class: 'icon', alt: 'group_icon') %>
+      <div class="group_info">
+        <span class="name"><%= @group.name %></span>
+        
+        <span class="date"><%= @group.created_at.strftime('%d %b %Y') %></span>
+      </div>
+    <% end %>
+  </div>
+
+  <% if @purchases.empty? %>
+    <div class="empty">
+      <p>Group has no transactions.</p>
+    </div>
+  <% end %>
+
+  <ul class="purchase_list">
+    <% @purchases.each do |purchase| %>
+      <li>
+        <%= link_to group_purchase_url(@group, purchase), data: { turbo: false }, class: "item purchase" do %>
+          <div class="group_info no_margin">
+            <span class="name"><%= purchase.name %></span>
+            <span class="amount">$<%= purchase.amount %></span>
+            <div class="date">
+              <%= inline_svg_tag "calendar.svg", class: "calendar" %>
+              <%= purchase.created_at.strftime('%d %b %Y') %>
+            </div>
+          </div>
+        <% end %>
+        <ul class="item associated_categories">
+          <% purchase.groups.each do |group| %>
+            <li class="associated_category">
+              <%= image_tag(check_url(group.icon), class: 'icon', alt: 'group_icon') %>
+              <span><%= group.name %></span>
+            </li>
+          <% end %>
+        </ul>
+      </li>
+    <% end %>
+  </ul>
+  <div class="item checkout">
+    <span class="name">total expences:</span>
+    <span class="amount">$<%= total_amount(@group)%></span>
+  </div>
+</main>
```

Comment:
- [ ] @Zilola-Nazarova Great work on your transaction page, however the layout slightly differs with the given design, eg the placement of total expenses and transaction details below is a comparison:
![image](https://github.com/Zilola-Nazarova/budget-app/assets/61408860/4888a5d9-6352-4390-aa5c-291eaef93b99)


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Zilola-Nazarova/budget-app/assets/61408860/5b8c803d-ba71-42fb-8dbc-4362058923c4)  |  ![image](https://github.com/Zilola-Nazarova/budget-app/assets/61408860/67ba6c8e-ab0f-4bfa-b1f7-5edfc2d83542)